.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We'll want to put this file into .gitignore, right ?

Reply:
Whoops. Did put this in global .gitignore. But apparently got committed anyway.

Reply:
fixed now by recommitting changes